lib/taskana-core-test/src/test/java/acceptance/jobs/AbstractTaskanaJobAccTest.java

```diff
@@ -131,6 +133,37 @@ void should_FindNoJobsToRunUntilFirstRunIsReached_When_CleanupScheduleIsInitiali
     }
   }
 
+  @WithAccessId(user = "admin")
+  @Test
+  void should_DeleteOldHistoryCleanupJobs_When_InitializingSchedule() throws Exception {
```

Comment:
What do you think about combining this test together wicth should_DeleteOldTaskCleanupJobs_When_InitializingSchedule() using a @TestFactory?

Reply:
Yes I will do it